migrations/caravan_interrupts.lua

```diff
@@ -0,0 +1,48 @@
+for unit_number, caravan_data in pairs(storage.caravans or {}) do
+    caravan_data.interrupts = caravan_data.interrupts or {}
+
+    -- Increase fuel inventory sizes
+    local old_inventory = caravan_data.fuel_inventory
+    local new_inventory = game.create_inventory(#old_inventory * 2)
```

Comment:
This will cause issues. Caravans in previous versions had different inventory sizes than 1. I would recommend to instead import `caravan-prototypes.lua` from within the migration and set this value directly to `prototype.fuel_size`